packages/Webkul/Notification/src/Listeners/NotificationListener.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Webkul\Notification\Listeners;
+
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Webkul\Notification\Events\CreateNotification;
+use Webkul\Notification\Events\NotificationEventInterface;
+use Webkul\Notification\Models\Notification;
+
+class NotificationListener implements ShouldQueue
+{
+    /**
+     * Handle the event.
+     *
+     * @return void
+     */
+    public function handle(NotificationEventInterface $event)
+    {
+        $notification = Notification::create([
+            'type'         => $event->notificationData['type'],
+            'route'        => $event->notificationData['route'] ?? null,
+            'route_params' => $event->notificationData['route_params'] ?? null,
+            'title'        => $event->notificationData['title'] ?? null,
+            'description'  => $event->notificationData['description'] ?? null,
+            'context'      => $event->notificationData['context'] ?? null,
+        ]);
+
+        $userNotificationsData = collect($event->notificationData['user_ids'])->map(function ($userId) {
+            return [
+                'admin_id' => $userId,
+            ];
+        })->toArray();
```

Comment:
Couldn't 'array_map' have been used here instead of using collect to convert an array to collection then map and then converting back to an array.